src/mono/mono.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+    This project is not really a .NET package. It's just defined as a csproj to simplify
+    the build config. In reality the output assembly is empty and ignored.
```

Comment:
I converted `mono.targets` to `mono.csproj` because it simplifies the incremental build.